### DISCUSSION TITLE
Use of iCloud

### DISCUSSION THREAD

**Discussion by devilgate** [vincode-io/Zavala#106] [Support] [answered]
I think I must be missing something, but I can't see how to create an outline in iCloud. I've worked out that you can drag an existing one _to_ iCloud, as long as you hover over the 'All' section and wait for it to light up.

**Comment by vincode-io** [comment, marked as answer]
Just select the tag or 'All' in the iCloud section of the sidebar and hit the new feed button. Zavala will create your new Outline in that location.

**Reply by devilgate** [reply]
Huh. I was sure I'd tried that and it didn't work. Now it does. Thanks.
